plugin/rollback.go

```diff
@@ -219,59 +244,68 @@ func (b *backend) serviceAccountPolicyRollback(ctx context.Context, req *logical
 	return err
 }
 
-func (b *backend) deleteServiceAccount(ctx context.Context, iamAdmin *iam.Service, account gcputil.ServiceAccountId) error {
-	if account.EmailOrId == "" {
-		return nil
+func (b *backend) serviceAccountPolicyDiffRollback(ctx context.Context, req *logical.Request, data interface{}) error {
+	b.rolesetLock.Lock()
```

Comment:
I'm wondering if this is the correct lock to use. WAL entries of type `walTypeIamPolicyDiff` are only used for static accounts, which use the backend's `staticAccountLock`.

Reply:
I've changed this to `staticAccountLock`.